examples/count-linear-extensions-using-hpolytope/volesti_lecount.cpp

```diff
@@ -0,0 +1,220 @@
+#include "Eigen/Eigen"
+#include <vector>
+#include "cartesian_geom/cartesian_kernel.h"
+#include "hpolytope.h"
+#include "known_polytope_generators.h"
+
+#include "random_walks/random_walks.hpp"
+
+#include "volume_sequence_of_balls.hpp"
+#include "volume_cooling_gaussians.hpp"
+#include "volume_cooling_balls.hpp"
+
+#include "preprocess/max_inscribed_ellipsoid_rounding.hpp"
+#include "preprocess/min_sampling_covering_ellipsoid_rounding.hpp"
+#include "preprocess/svd_rounding.hpp"
+
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include "misc.h"
+
+
+enum VOL_OPTIONS {
+	SOB,
+	CG,
+	CB
+};
+
+enum ROUND_OPTIONS {
+	SVD,
+	MIN_ELLIPSOID
+};
+
+
+typedef double NT;
+typedef Cartesian <NT> Kernel;
+typedef typename Kernel::Point Point;
+typedef BoostRandomNumberGenerator<boost::mt19937, NT, 5> RNGType;
+typedef HPolytope <Point> HPOLYTOPE;
+typedef typename HPOLYTOPE::MT MT;
+typedef typename HPOLYTOPE::VT VT;
+
+struct ArgOptions {
+	VOL_OPTIONS vo;
```

Comment:
please use a 4 spaces indentation as in other files of the library

Reply:
Done :+1: